aten/src/ATen/native/cuda/SummaryOps.cu

```diff
@@ -32,7 +32,9 @@ __global__ void kernelHistogram1D(
     detail::TensorInfo<output_t, IndexType> a, /* output */
     detail::TensorInfo<output_t, IndexType> p, /* partial output */
     detail::TensorInfo<input_t, IndexType> b, /* input */
-    int binsize,
+    scalar_t minvalue,
+    scalar_t maxvalue,
```

Comment:
Does this kernel need all three of `{minvalue, maxvalue, binsize}`? Shouldn't `binsize` be enough to implement both histc and bincount?

Reply:
for histc, I thought we need to check if data is within the provided min/max? Also, min is used to calculate the bin number. for bincount, neither is needed

Reply:
Got it, thank you for the clarification/reminder. Yes, you are right